modules/ossm-security-cert-manage.adoc

```diff
@@ -44,105 +44,78 @@ spec:
+$ sed -n '/-----BEGIN CERTIFICATE-----/{:start /-----END CERTIFICATE-----/!{N;b start};/.*/p}' bookinfo-proxy-cert.txt > certs.pem
+$ awk 'BEGIN {counter=0;} /BEGIN CERT/{counter++} { print > "proxy-cert-" counter ".pem"}' < certs.pem
 ----
 +
-The file `/tmp/pod-cert-chain.pem` contains the workload certificate and the CA certificate propagated to the pod.
-+
-. Verify the root certificate is the same as the one specified by the Operator. Replace `<path>` with the path to your certificates.
+You should have three files in your working dir: `proxy-cert-1.pem`, `proxy-cert-2.pem` and `proxy-cert-3.pem`.
+
+. Verify that the root certificate is the same as the one specified by the administrator. Replace `<path>` with the path to your certificates.
 +
 [source,terminal]
 ----
 $ openssl x509 -in <path>/root-cert.pem -text -noout > /tmp/root-cert.crt.txt
+$ openssl x509 -in ./proxy-cert-3.pem -text -noout > /tmp/pod-root-cert.crt.txt
+$ diff -s /tmp/root-cert.crt.txt /tmp/pod-root-cert.crt.txt
+Files /tmp/root-cert.crt.txt and /tmp/pod-root-cert.crt.txt are identical
 ----
+
+. Verify that the CA certificate is the same as the one specified by the administrator. Replace `<path>` with the path to your certificates.
 +
 [source,terminal]
 ----
-$ openssl x509 -in /tmp/pod-root-cert.pem -text -noout > /tmp/pod-root-cert.crt.txt
-----
-+
-[source,terminal]
-----
-$ diff /tmp/root-cert.crt.txt /tmp/pod-root-cert.crt.txt
-----
-+
-Expect the output to be empty.
-+
-. Verify the CA certificate is the same as the one specified by Operator. Replace `<path>` with the path to your certificates.
-+
-[source,terminal]
-----
-$ sed '0,/^-----END CERTIFICATE-----/d' /tmp/pod-cert-chain.pem > /tmp/pod-cert-chain-ca.pem
-----
-+
-[source,terminal]
-----
-$ openssl x509 -in <path>/ca-cert.pem -text -noout > /tmp/ca-cert.crt.txt
-----
-+
-[source,terminal]
-----
-$ openssl x509 -in /tmp/pod-cert-chain-ca.pem -text -noout > /tmp/pod-cert-chain-ca.crt.txt
-----
-+
-[source,terminal]
-----
-$ diff /tmp/ca-cert.crt.txt /tmp/pod-cert-chain-ca.crt.txt
+$ openssl x509 -in ./proxy-cert-2.pem -text -noout > /tmp/pod-cert-chain-ca.crt.txt
+$ diff -s /tmp/ca-cert.crt.txt /tmp/pod-cert-chain-ca.crt.txt
+Files /tmp/ca-cert.crt.txt and /tmp/pod-cert-chain-ca.crt.txt are identical
 ----
-+
-Expect the output to be empty.
-+
+
 . Verify the certificate chain from the root certificate to the workload certificate. Replace `<path>` with the path to your certificates.
 +
 [source,terminal]
 ----
-$ head -n 21 /tmp/pod-cert-chain.pem > /tmp/pod-cert-chain-workload.pem
-----
-+
-[source,terminal]
-----
-$ openssl verify -CAfile <(cat <path>/ca-cert.pem <path>/root-cert.pem) /tmp/pod-cert-chain-workload.pem
-----
-+
-.Example output
-[source,terminal]
-----
-/tmp/pod-cert-chain-workload.pem: OK
+$ openssl verify -CAfile <(cat <path>/ca-cert.pem <path>/root-cert.pem) ./proxy-cert-1.pem
+./proxy-cert-1.pem: OK
 ----
 
+
```

Comment:
Are these blank lines intentional?

Reply:
Thanks for pointing this out. I submitted a revision to clean it up.